code/modules/mob/living/carbon/human/human_species.dm

```diff
@@ -11,11 +11,18 @@
 /mob/living/carbon/human/dummy/mannequin/Initialize()
 	. = ..()
 	GLOB.human_mob_list -= src
+	GLOB.living_mob_list -= src
+	GLOB.player_list -= src
```

Comment:
you need to clean your branches

Reply:
Fun, that wasn't even part of the commit, and already exists as part of MonkEris.